tests/test_commands.py

```diff
@@ -225,10 +215,33 @@ def test_bigchain_run_init_when_db_exists(mock_db_init_with_existing_db):
     run_init(args)
 
 
-def test_drop_existing_db(mock_rethink_db_drop):
+@patch('bigchaindb.backend.schema.drop_database')
+def test_drop_db_when_assumed_yes(mock_db_drop):
```

Comment:
These drop db interactive tests were (more or less) previously done in the `db/test_utils.py` file that was moved to `db/rethinkdb/test_schema.py`.